ros/angel_system_nodes/angel_system_nodes/spatial_mapper.py

```diff
@@ -196,9 +196,9 @@ def detection_callback(self, detection):
 
             # get pixel positions of detected object box and form into box corners
             min_vertex0 = detection.left[i]
-            min_vertex1 = detection.top[i]
+            min_vertex1 = detection.bottom[i]
```

Comment:
It seems like we have some clarity debt.

[A] We should add a comment about what the intended screen-space order of `corners_screen_pos` is, extending the comment on line 197.

[B] In terms of values being assigned to variables, it confused me that the "bottom" was being assigned to a "min" vertex, as "bottom" in screen space is the maximal value. It would have made more sense to me to leave the vertex variable assignment the same and change the assignment to `corners_screen_pos` instead.

[C] Not for this PR, but it seems we might want to change the names of the slots in the detection 3d set message as well. The cardinal direction naming presents a bundled assumption that, in screen-space (2D), the box is axis-aligned. That is not the case in the box resulting from the 3D transformation (*not* axis aligned in 3D), so those names should be expanded on in a revision.

Reply:
Should i just remove this from this PR and open a new one?

Reply:
If this specific change is a required bug fix for the near term, then leave it here, but if not then an expanded "real" fix in a separate PR seems appropriate.

Reply:
I opened an issue just so we don't forget about this: https://github.com/PTG-Kitware/angel_system/issues/60